packages/components/package.json

```diff
@@ -65,6 +65,8 @@
   "dependencies": {
     "@giscus/react": "3.0.0",
     "@next/bundle-analyzer": "14.2.13",
+    "@radix-ui/react-navigation-menu": "^1.2.0",
+    "@theguild/tailwind-config": "0.5.0",
```

Comment:
why do we need it now as a dependency?

Reply:
We can make it a peer dep or implicit, but we do depend on the colors and scales defined in the config.
I think I installed it to work around some TS error that shouldn't be there, either in Tailwind config, or the storybook file, but it is in fact a dependency, even if we didn't import it — a breaking change there, like removing a color, would break the components package and website downstream.